Alchemy/DirectXUtil/CG32bitImage.cpp

```diff
@@ -904,92 +905,8 @@ void CG32bitImage::DrawDot (int x, int y, CG32bitPixel rgbColor, MarkerTypes iMa
 
 			break;
 			}
-
-		case markerSmallCross:
-			{
-			int yTop = y - SMALL_SQUARE_SIZE;
-			int yBottom = y + SMALL_SQUARE_SIZE;
-			int xLeft = x - SMALL_SQUARE_SIZE;
-			int xRight = x + SMALL_SQUARE_SIZE;
-
-			//	Draw the horizontal line
-
-			if (y >= m_rcClip.top && y < m_rcClip.bottom)
-				{
-				CG32bitPixel *pRowStart = GetPixelPos(0, y);
-				CG32bitPixel *pPos = pRowStart + Max((int)m_rcClip.left, xLeft);
-				CG32bitPixel *pPosEnd = pRowStart + Min(xRight+1, (int)m_rcClip.right);
-
-				while (pPos < pPosEnd)
-					*pPos++ = rgbColor;
-				}
-
-			//	Draw the vertical line
-
-			if (x >= m_rcClip.left && x < m_rcClip.right)
-				{
-				CG32bitPixel *pPos = GetPixelPos(x, Max(yTop+1, (int)m_rcClip.top));
-				CG32bitPixel *pPosEnd = GetPixelPos(x, Min(yBottom, (int)m_rcClip.bottom));
-
-				while (pPos < pPosEnd)
-					{
-					*pPos = rgbColor;
-					pPos = NextRow(pPos);
-					}
-				}
-
-			break;
-			}
-
-		case markerMediumCross:
-			{
-			int yTop = y - MEDIUM_SQUARE_SIZE;
-			int yBottom = y + MEDIUM_SQUARE_SIZE;
-			int xLeft = x - MEDIUM_SQUARE_SIZE;
-			int xRight = x + MEDIUM_SQUARE_SIZE;
-
-			//	Draw the horizontal line
-
-			if (y >= m_rcClip.top && y < m_rcClip.bottom)
-				{
-				CG32bitPixel *pRowStart = GetPixelPos(0, y);
-				CG32bitPixel *pPos = pRowStart + Max((int)m_rcClip.left, xLeft);
-				CG32bitPixel *pPosEnd = pRowStart + Min(xRight+1, (int)m_rcClip.right);
-
-				while (pPos < pPosEnd)
-					*pPos++ = rgbColor;
-				}
-
-			//	Draw the vertical line
-
-			if (x >= m_rcClip.left && x < m_rcClip.right)
-				{
-				CG32bitPixel *pPos = GetPixelPos(x, Max(yTop, (int)m_rcClip.top));
-				CG32bitPixel *pPosEnd = GetPixelPos(x, Min(yBottom+1, (int)m_rcClip.bottom));
-
-				while (pPos < pPosEnd)
-					{
-					*pPos = rgbColor;
-					pPos = NextRow(pPos);
-					}
-				}
-
-			break;
-			}
-
-		case markerSmallFilledSquare:
-			{
-			int xLeft = x - SMALL_SQUARE_SIZE;
-			int yTop = y - SMALL_SQUARE_SIZE;
-			CG32bitPixel rgbInsideColor = CG32bitPixel::Blend(0, rgbColor, (BYTE)192);
-
-			DrawDot(x, y, rgbColor, markerSmallSquare);
-			Fill(xLeft + 1, yTop + 1, 2 * SMALL_SQUARE_SIZE - 1, 2 * SMALL_SQUARE_SIZE - 1, rgbInsideColor);
-			break;
-			}
-
 		case markerSmallCircle:
-			{
```

Comment:
In general, please adhere to Whitesmith indenting style. I know it is unpopular, but the whole source code uses it, so I don't want some parts to be different.
See: https://en.wikipedia.org/wiki/Indentation_style#Whitesmiths_style

Reply:
VS2019 keeps breaking the indention and using other things by default - is there any way to get it to use that indent style, or an automated tool to fix it across the whole project?

Reply:
Fixed a whole bunch in a recent commit manually, let me know if theres anything else to fix.